velox/common/memory/MemoryPool.cpp

```diff
@@ -543,6 +542,9 @@ void MemoryPoolImpl::allocateContiguous(
     MachinePageCount numPages,
     ContiguousAllocation& out) {
   CHECK_AND_INC_MEM_OP_STATS(Allocs);
+  if (!out.empty()) {
+    CHECK_AND_INC_MEM_OP_STATS(Frees);
```

Comment:
Since we have passed the check above, how about we add a method say
```
INC_MEM_OP_STATS to update stats only here?

  if (!out.empty()) {
    INC_MEM_OP_STATS(Frees);
  }
```